shape_msgs/Mesh[]  meshes
geometry_msgs/PoseStamped[]  poses

---
uint8 RESULT_SUCCESS            = 0
uint8 RESULT_INVALID_ID         = 1
uint8 RESULT_ROBOT_IN_COLLISION = 2

uint8 result
